301-web-app-sql-docdb-search/azuredeploy.json

```diff
@@ -351,7 +351,7 @@
     {
       "name": "appinsights",
       "type": "Microsoft.Insights/components",
```

Comment:
Microsoft.Insights is not globally available, so you do need to restrict the locations that can be used here - either by adding another parameter for Insights location or using a map based on the passed in location.

Reply:
Extracted parameter for insights location and set for both clouds